dpgen2/entrypoint/submit.py

```
@@ -81,6 +81,7 @@
     CustomizedLmpTemplateTaskGroup,
     ExplorationStage,
     ExplorationTask,
+    LmpSpinTaskGroup,
```

Ruff (0.11.9)

`dpgen2.exploration.task.LmpSpinTaskGroup` imported but unused

Remove unused import

(F401)